compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -3353,7 +3314,6 @@ declare_lint_pass! {
         INDIRECT_STRUCTURAL_MATCH,
         INEFFECTIVE_UNSTABLE_TRAIT_IMPL,
         INLINE_NO_SANITIZE,
-        INVALID_ALIGNMENT,
```

Comment:
Should this be added to the `register_removed` list?

Reply:
Ah, I guess it should, though considering that we never actually emitted the lint it probably doesn't make a difference.